datamonster_api/tests/lib/regression.py

```diff
@@ -143,24 +223,22 @@ def test_data_source_2():
     assert_frame_equal(df.head(1), pandas.DataFrame.from_records([records]))
 
 
-def test_bigger_data_source():
-    """ Test SimilarWeb which is a bigger dataset
+def test_get_data_bigger():
+    """ Test `SimilarWeb Direct Volume` which is a bigger dataset
     """
     company = dm.get_company_by_id(335)
-    agg = Aggregation(period="week", company=company)
-
     ds = dm.get_datasource_by_id("5899e237-874c-4e77-9d2e-c4b6cff218e8")
-    assert ds.name == "SimilarWeb Direct Volume"
```

Comment:
Why aren't we checking the name?

Reply:
the point here was just to check that we can pull in a larger data set - the name property is checked on the QA ETL (and is repetitive/annoying to check all the time)

Reply:
kk